.env

```diff
@@ -1,5 +1,7 @@
 REACT_APP_TILECACHE_URL=https://tiles.pacificclimate.org/tilecache/tilecache.py
-REACT_APP_NCWMS_URL=https://services.pacificclimate.org/pcex/ncwms
+REACT_APP_NCWMS_URL=http://docker-dev02.pcic.uvic.ca:30778/ncWMS/wms
```

Comment:
It depends on what you think the role of these `.env` files should be. Matthew _always_ overrides them, and that is pretty much the way it should/must be ... there can and now will (with our staging turnover) be multiple deployments of the same apps with different configurations to reflect that. So my opinion right now is that these should be set to whatever is convenient for the developer, since that is who they will serve. What those values should be is perhaps up for grabs.